snapcraft/cli/snapcraftctl/_runner.py

```diff
@@ -0,0 +1,80 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import functools
+import json
+import logging
+import os
+import sys
+
+import click
+
+from snapcraft.internal.errors import SnapcraftEnvironmentError
+from snapcraft.cli._errors import exception_handler
+from snapcraft.internal import log
+
+
+@click.group()
+@click.option('--debug', '-d', is_flag=True)
```

Comment:
it would be nice if this could be picked up from the environment as well `SNAPCRAFT_DEBUG` (click supports this) so then it is hooked up with snapcraft's flag too

Reply:
Oh good call, fixed in the most recent commit.